src/main/features/offer/views/defendant-response.njk

```diff
@@ -0,0 +1,69 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, submitButton, errorSummary, radioGroup %}
+{% from "date.njk" import dateInput %}
+
+{% set title = t('Money claim') %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <h1 class="heading-large form-hint">{{ t("Defendant's response") }} </h1>
+      <form novalidate method="post">
+        {{ csrfProtection(csrf) }}
+        <div class="description-block">
+          <h2 class="heading-large"> 
+          {{ t('%s’s offer',{
+              postProcess: 'sprintf',
+              sprintf: [claim.claimData.defendant.name]
+            }) }}</h2>
+        </div>
+        <table class="claim-dashboard">
```

Comment:
That does not make sense to have it here. Please remove or reimplement for an offer table.